cryptlex/lexfloatclient/lexfloatclient.py

```diff
@@ -267,6 +267,28 @@ def GetFloatingClientMeterAttributeUses(name):
         else:
             raise LexFloatClientException(status)
 
+    @staticmethod
+    def GetFloatingClientMetadata(key):
+        """Gets the value of the floating client metadata.
+
+        Args:
+                key (str): metadata key to retrieve the value
+
+        Raises:
+                LexFloatClientException
+
+        Returns:
+                str: value of metadata for the key
+        """
```

Comment:
@muneebkq Update the docs, it does not return the value of metadata for the key, instead it simply returns `value of the floating client metadata`

Reply:
added